services/datamanager/datasync/upload_data_capture_file.go

```diff
@@ -162,19 +163,18 @@ func sendStreamingDCRequests(ctx context.Context, stream v1.DataSyncService_Stre
 }
 
 func getFileExtFromImageFormat(res pb.Format) string {
-	defaultFileExt := ""
 	switch res {
 	case pb.Format_FORMAT_JPEG:
-		return ".jpeg"
+		return utils.MimeTypeJPEG
 	case pb.Format_FORMAT_PNG:
-		return ".png"
+		return utils.MimeTypePNG
 	case pb.Format_FORMAT_RAW_DEPTH:
-		return ".dep"
+		return utils.MimeTypeRawDepth
```

Comment:
We do want these to be the file extensions, since we use them to infer the mime type in app [here](https://github.com/viamrobotics/app/blob/87061255436b743165ce2ed6c47fdf4e9e5888d1/datasync/server.go#L135-L152). We should actually likely just upload the mime type explicitly from RDK, but that would definitely be out of scope here.